src/helper.h

```diff
@@ -99,6 +99,12 @@ class Client: public RefCountable
     /// \param madeProgress whether the died helper(s) responded to any requests
     void handleFewerServers(bool madeProgress);
 
+    /// creates helper processes
+    virtual void openServers();
+
+    /// handles exited helper process
+    void serverClosed(SessionBase *);
```

Comment:
Unless I am missing something, this method should take a reference and avoid legacy terminology:

```suggestion
    void sessionClosed(SessionBase &);
```